matchbook-sdk-rest/src/main/java/com/matchbook/sdk/rest/HttpConfig.java

```diff
@@ -35,21 +37,21 @@ public String toString() {
 
     public static class Builder {
 
-        private int connectionTimeout = 15_000;
-        private int writeTimeout = 15_000;
-        private int readTimeout = 15_000;
+        private long connectionTimeout = TimeUnit.SECONDS.toMillis(5);
+        private long writeTimeout = TimeUnit.SECONDS.toMillis(15);
+        private long readTimeout = TimeUnit.SECONDS.toMillis(30);
```

Comment:
why do you think we should change those numbers?

Reply:
15 seconds for each timeout looked weird to me, but I don't like my numbers too. What about 2 seconds for connection and 10 seconds for read/write operations as a default?

Reply:
a default timeout for connectiontimeout, writetimeout and readtimeout is 10 seconds, we can use them atm, and leave the user to set the number he wants thought ClientConfig properties.

Reply:
Deal :slightly_smiling_face:

Reply:
I used 10 secs timeout for everything.